### ISSUE TITLE
Label right tree in multiple cophylo/tanglegrams with ggtree

### ISSUE THREAD

**Issue by dandaman** [YuLab-SMU/ggtree#502]
Hi,

I experience a strange behaviour with ggtree (see my sessionInfo below). 

I'd like to do a multiple cophylo plot like you show in [chapter 2](https://yulab-smu.top/treedata-book/chapter2.html#ggtree-fortify): 
![](https://yulab-smu.top/treedata-book/treedata_files/figure-html/cophylo3-1.svg?raw=true)

If I use the code from the two code blocks in my plot always the left-most tree is labelled (geom_tiplab; See the attached png, jupyter notebook and Conda environment). Strangely I have done this before and it worked nicely. I tried using the same ggtree version (2.4.0 - now its 3.0.2) than that time but still get the same result.

Am I doing something wrong? Is there a way to choose the tree to label with geom_tiplab besides `data=x` ?
Or is this a bug?

Thank you and all the best,
Daniel

```R
sessionInfo()
R version 4.1.3 (2022-03-10)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: Ubuntu 20.04.4 LTS

Matrix products: default
BLAS/LAPACK: /mnt/soft/miniconda3/envs/dl_cophylo/lib/libopenblasp-r0.3.20.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] ggplot2_3.3.3 ggtree_3.0.2  dplyr_1.0.6

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.6          pillar_1.6.1        compiler_4.1.3
 [4] BiocManager_1.30.15 aplot_0.0.6         jsonlite_1.7.2
 [7] tidytree_0.3.4      lifecycle_1.0.0     tibble_3.1.2
[10] nlme_3.1-157        gtable_0.3.0        lattice_0.20-45
[13] pkgconfig_2.0.3     rlang_0.4.11        DBI_1.1.1
[16] rvcheck_0.1.8       patchwork_1.1.1     parallel_4.1.3
[19] treeio_1.16.1       withr_2.4.2         generics_0.1.0
[22] vctrs_0.3.8         grid_4.1.3          tidyselect_1.1.1
[25] glue_1.4.2          R6_2.5.0            fansi_0.4.2
[28] purrr_0.3.4         tidyr_1.1.3         magrittr_2.0.1
[31] scales_1.1.1        ellipsis_0.3.2      assertthat_0.2.1
[34] ape_5.5             colorspace_2.0-1    utf8_1.2.1
[37] lazyeval_0.2.2      munsell_0.5.0       crayon_1.4.1
```

**Comment by dandaman** [comment]
![index](https://user-images.githubusercontent.com/979208/169019895-c4ff8127-ce41-45e8-9679-79c7961ebab2.png)

**Comment by dandaman** [comment]
[Daniel.zip](https://github.com/YuLab-SMU/ggtree/files/8715496/Daniel.zip)

**Comment by xiangpin** [comment]
Please update `ggtree`. It has been fixed in the latest released version (Now it is `3.4.0` in [Bioc 3.15](https://www.bioconductor.org/packages/release/bioc/html/ggtree.html)).

**Comment by dandaman** [comment]
Thank you - This worked for me 👍
